### ISSUE TITLE
Scalafmt setup throws `NoSuchElementException`

### ISSUE THREAD

**Issue by jvican** [lloydmeta/enumeratum#99]
When cloning the repo and running sbt with ` +benchmarking/jmh:run -i 10 -wi 10 -f3 -t 1`, I get the following:

```
> last benchmarking/compile:scalafmt
java.lang.NoSuchMethodError: scala.Option.contains(Ljava/lang/Object;)Z
	at org.scalafmt.cli.StyleCache$.getStyleForFileOrError(StyleCache.scala:25)
	at org.scalafmt.cli.Scalafmt210.format(Scalafmt210.scala:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.scalafmt.sbt.HasScalaFmt.org$scalafmt$sbt$HasScalaFmt$$handleFile(HasScalaFmt.scala:85)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at org.scalafmt.sbt.HasScalaFmt$$anonfun$writeFormattedContentsToFiles$1$$anonfun$apply$3.apply(HasScalaFmt.scala:54)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.parallel.immutable.ParHashSet$ParHashSetIterator.foreach(ParHashSet.scala:76)
	at scala.collection.parallel.ParIterableLike$Foreach.leaf(ParIterableLike.scala:975)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply$mcV$sp(Tasks.scala:54)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$$anonfun$tryLeaf$1.apply(Tasks.scala:53)
	at scala.collection.parallel.Task$class.tryLeaf(Tasks.scala:56)
	at scala.collection.parallel.ParIterableLike$Foreach.tryLeaf(ParIterableLike.scala:972)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.internal(Tasks.scala:172)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.internal(Tasks.scala:514)
	at scala.collection.parallel.AdaptiveWorkStealingTasks$WrappedTask$class.compute(Tasks.scala:162)
	at scala.collection.parallel.AdaptiveWorkStealingForkJoinTasks$WrappedTask.compute(Tasks.scala:514)
	at scala.concurrent.forkjoin.RecursiveAction.exec(RecursiveAction.java:160)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
[error] (benchmarking/compile:scalafmt) java.lang.NoSuchMethodError: scala.Option.contains(Ljava/lang/Object;)Z
```
  
I believe there is a problem with the setup of scalafmt. Can someone else confirm?

**Comment by lloydmeta** [comment]
I hate to say it, but "works fine here". Do you happen to have Scalafmt installed as a global plugin as well?

**Comment by jvican** [comment]
It is indeed very weird... I have scalafmt installed globally but I don't see why that should affect the setup since sbt is using the jar from the maven cache. It's not a blocker for me, so don't waste time on this for now :wink:.

**Comment by lloydmeta** [comment]
Yeah, no idea, it was just a stab in the dark. I think it's no longer the norm to have scalafmt as a plugin in a project since it's no longer supported, so I thought it might be related, but that error is very strange.

**Comment by jvican** [comment]
Yes, I also think that was deprecated. I think there's another way to integrate scalafmt into sbt, but only available on 0.13.13.
